test/options_test.rb

```diff
@@ -88,5 +88,9 @@ module Slop; class BarOption < Option; end; end
       @options_config = Slop::Options.new({banner: "custom banner"})
       assert_match(/^custom banner/, @options_config.to_s) 
     end
+    it "banner is disabled" do
+      @options_config = Slop::Options.new({banner: false})
+      assert_match(//, @options_config.to_s)
```

Comment:
I don't think this is a good way to do the match, since `//` matches any string so it might have false positives.